source/mongodump.txt

```diff
@@ -648,11 +658,6 @@ Options
       - :method:`db.collection.renameCollection()`
       - :method:`db.collection.aggregate()` with :pipeline:`$out`
 
-   ``--oplog`` has no effect when running ``mongodump``
```

Comment:
We should keep this "--oplog has no effect when running mongodump against a mongos [...]" piece, and delete the "However, you can use ``--oplog`` to dump
   individual shards."